project-templates/ecs/basic/README.md

```diff
@@ -0,0 +1,161 @@
+![img.png](../readme-images/waypoint_tfc_creds_setup_ss.png)
+
+
+2. Create a Waypoint Project Template by filling in the fields in the Template Creation page:
+![img_1.png](../readme-images/waypoint_create_template_ss.png)
+
+
+
+3. Return to your Projects List Page. In the top right corner, select `Create Project`. From the dropdown menu, select `Create Project with template`:
+![img_3.png](../readme-images/waypoint_projects_list_ss.png)
+
+   
+4. Select the template you would like to use for this project.
+![img_4.png](../readme-images/waypoint_select_template_ss.png)
+ 
+   
+
+5. Fill in the name of your project under `Project Name`. Then click `Create Project.
+![img_6.png](../readme-images/waypoint_create_project_ss.png)
+
+   
+6. Review the project details of your newly created project! You're Ready to Go!
+![img_7.png](../readme-images/waypoint_project_details_ss.png)
+
+
+## Waypoint Template Steps via CURL commands
+
+Pre-Requisites
+1. Your NamespaceID
+2. HCP Waypoint Authentication Token
+3. TFC Authentication Token
+
+
+
+1. Set up your TFC Credentials in Waypoint
+
+```shell
+curl --location --request PUT 'https:// api.hashicorp.cloud:443/waypoint/2022-02-03/namespace/{{NAMESPACE_ID}}/tfcconfig' \
+--header 'Authorization: {{YOUR_HCP_Waypoint_AUTH_TOKEN}}' \
+--header 'Content-Type: application/json' \
+--data '{"tfc_config":{"organization_name":"hcp_waypoint_integration", "token":"{{YOUR_TFC_AUTH_TOKEN}}"}, "namespace_id":"{{NAMESPACE_ID}}"}'
+
+
+```
+
+2. Create a Waypoint Project Template
+
+```shell
+curl --location 'https:// api.hashicorp.cloud:443/waypoint/2022-02-03/namespace/{{NAMESPACE_ID}}/projecttemplate' \
+--header 'Authorization: {{YOUR_HCP_Waypoint_AUTH_TOKEN}}' \
+--header 'Content-Type: application/json' \
+--data '{
+  "project_template": {
+    "name": "{{SOME_TEMPLATE_NAME}}",
+    "summary": "{{SOME_TEMPLATE_SUMMARY}}",
+    "readme_markdown_template": "{{SOME_TEMPLATE_README_TEXT}}",
+    "waypoint_project": {
+      "waypoint_hcl_template": "{{ENCODED_PROJECT_TEMPLATE_BYTES}}"
+    },
+    "terraform_nocode_module": {
+      "source": "private/hcp_waypoint_integration/example-microservice-ecs-allenvs/aws",
+      "version": "0.1.2"
+    }
+  }
+}'
+```
+
+
+3. Initialize a Waypoint Project Using the Existing Project Template.
+
+```shell
+curl --location 'https:// api.hashicorp.cloud:443/waypoint/2022-02-03/namespace/{{NAMESPACE_ID}}/project/fromtemplate' \
+--header 'Authorization: Bearer {{YOUR_HCP_Waypoint_AUTH_TOKEN}}' \
+--header 'Content-Type: application/json' \
+--data '{
+  "project_name": "{{SOME_HCP_WAYPOINT_PROJECT_NAME}}",
+  "project_template": {
+    "name": "{{PROJECT_TEMPLATE_NAME}}"
+  }
+}'
+```
+
+
+4. List all Project Templates
+```shell
+curl --location 'https:// api.hashicorp.cloud:443/waypoint/2022-02-03/namespace/{{NAMESPACE_ID}}/projecttemplates' \
```

Comment:
```suggestion
curl --location 'https:// api.hashicorp.cloud:443/waypoint/2022-02-03/namespace/{{NAMESPACE_ID}}/project-templates' \
```

Reply:
Recently changed in https://github.com/hashicorp/cloud-waypoint-service/pull/591